### ISSUE TITLE
[v1.36.0] Advanced examples no longer working

### ISSUE THREAD

**Issue by kimthompson** [mobile-dev-inc/maestro#1696]
**Describe the bug**
I'm trying out Maestro Cloud, and was surprised that my flow failed immediately because it couldn't find one of my subflows. I downloaded the examples to see what I might be missing, and I discovered that those flows also didn't know what to do with subflows.

**To Reproduce**
Simply follow [your own walkthrough](https://cloud.mobile.dev/getting-started/quickstart):

1. Install/update the CLI with `curl -Ls "https://get.maestro.mobile.dev" | bash`
2. Log in with `maestro login`
3. Download the samples with `maestro download-samples`
4. Try to run one of the advanced flows from the 'samples' folder: `maestro cloud sample.apk android-advanced-flow.yaml`
5. See an error message:

![Screenshot 2024-03-14 at 5 18 54 PM](https://github.com/mobile-dev-inc/maestro/assets/3376722/038b25ce-0bc0-489f-ba8d-bcf4758d9620)

This also applies to the advanced iOS tests run with `maestro cloud sample.zip ios-advanced-flow.yaml`.

![Screenshot 2024-03-14 at 5 19 47 PM](https://github.com/mobile-dev-inc/maestro/assets/3376722/d3e14776-5efd-44ec-b608-db64c9ed985f)

**Expected behavior**
The example test cases should run. More broadly, you should be able to refer to and run other Maestro flows within the same directory and within child directories.

**Comment by axelniklasson** [comment]
@kimthompson I understand why this is confusing for you - the walkthrough shows you how to run the simple Flows, whereas the advanced Flows require you to upload the full `samples` folder since just uploading `ios-advanced-flow.yaml` means that the subflows that that Flow is referring to is not part of the upload. I recognize that the error message is not the best here and I updated the docs to hopefully make this more clear.

So, to summarize - running `maestro cloud samples/sample.zip samples` should work for you.
